cmd/ipfs-cluster-ctl/formatters.go

```diff
@@ -204,8 +210,7 @@ func textFormatPrintAddedOutputQuiet(obj *addedOutputQuiet) {
 }
 
 func textFormatPrintMetric(obj *api.Metric) {
-	date := time.Unix(0, obj.Expire).UTC().Format(time.RFC3339)
-	fmt.Printf("%s: %s | Expire: %s\n", peer.IDB58Encode(obj.Peer), obj.Value, date)
+	fmt.Printf("%s: %s | Expires in: %s\n", peer.IDB58Encode(obj.Peer), obj.Value, time.Until(time.Unix(0, obj.Expire)).String())
```

Comment:
I was going to open an extra issue but let's fix it here:

`ipfs-cluster-ctl heath metrics freespace` should display the value in human form. We should also write the metric type (for all metrics): `<peer id> | freespace: 25GB | ...`. Also for ping we should write `<peer id> | ping | ...` (without an empty `: `).

There is a go-humanize library to do this (we already use it in sharding).